src/v/redpanda/admin/server.h

```diff
@@ -610,6 +612,9 @@ class admin_server {
       delete_transform(std::unique_ptr<ss::http::request>);
     ss::future<ss::json::json_return_type>
       list_committed_offsets(std::unique_ptr<ss::http::request>);
+    ss::future<ss::json::json_return_type> get_node_uuid_handler();
```

Comment:
nit: method declarations are placed in a way that makes it look like these are transform-related APIs